docs/INSTALLATION.md

```diff
@@ -1,31 +1,120 @@
+can continue as if everything is fine; we won't be able to provide support when
+something unexpected happens.
+
+### If node is not installed
+
+There are a couple of ways to install [NodeJS][web-nodejs]:
+
+* via an [Installer or Binary][web-nodejs-download]
+* via a [package manager][web-nodejs-package]
+
+Both options support Windows, MacOS, and Linux. If you don't know what to do,
+using an installer is the easier.
+
+- We recommend using the **LTS** version. This is also indicated as _recommended_
+on the [NodeJS][web-nodejs] website "for most users". 
+- Follow the instructions on the webpage and/or during the installer and install
+[NodeJS][web-nodejs].
 
-Each assignment needs some tools to run the tests:
+### Testing the installation
 
-* [Jest](https://facebook.github.io/jest/): a test runner based on Jasmine
-* [Babel](https://github.com/babel/babel): to transpile ECMAScript
-2015 code to ECMAScript 5
-* [ESLint](http://eslint.org/) (optional): to perform several static analysis
-and coding style checks to your JavaScript code.
+After the installer is done, or the package manager has completed, or the binary
+has been copied and the instructions have been followed, it's good to test if
+everything is alright.
+
+1. Open up a _terminal_ (`Terminal`, `cmd`,  `Powershell`, `bash`, ...)
+1. `node -v`
+
+The version should match the one on the website.
+
+**Note**: It is important you open a _new_ terminal window. Any open terminal
+windows might not have been refreshed after the installation completed. This
+means that the open terminals don't know that a new program was installed.
+
+> _**Help**_: `'node' is not recognised`
+>
+> If you've used the official installer, your `PATH` should have been
+> automatically configured, but if your shell has trouble locating your globally
+> installed modules &mdash; or if you build Node.js from source &mdash; update 
+> your `PATH` to include the `npm` binaries.
+>
+> On MacOS and Linux you may accomplish this by adding the following to either
+> `~/.bash_profile` or `~/.zshrc`:
+> 
+> ```bash
+> $ export PATH=/usr/local/share/npm/bin:$PATH
+> ```
+>
+> On Windows open the start menu and search for "environment variables". You'll
+> need to edit the `PATH` entry and _add_ the path to the `npm` binaries here.
+> Usually these are found at `C:\Program Files\nodejs`. If you browse here with
+> your `File Explorer`, you should find `node.exe`, `npm.bat` and `npx.bat`.
+>
+>
+> Close any open terminals and open a new one.
+
+# Assignment Requirements
+
+The `hello-world` assignment will teach you how to use the `exercism` cli and
+download an assignment to your computer. 
+
+Each assignment then needs some tools to run the tests:
+
+* [Jest][web-jest]: a test runner based on Jasmine
+* [Babel][web-babel]: to transpile ESNext code to work on your NodeJS version
+* [ESLint][web-eslint]: to perform several static analysis and coding style 
+checks to your JavaScript code.
```

Comment:
It might be a little intimidating for beginners of the track with all these tools being used. Do we need to mention these? Can we skip to `npm install`?

Reply:
I only have them because we had them before. So perfectly fine to _remove_ them here 🗡

Reply:
Updated!